File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.cpp
class Solution {
public:
    int percentageLetter(string s, char letter) {
             int count=0;
    for(int i=0;i<s.size();i++)
        {
            if(letter==s[i])
                {
                    count++;
                }
        }
        double p=count*100;
     return floor(p/s.size());
    }
};

File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.js
/**
 * @param {string} s
 * @param {character} letter
 * @return {number}
 */
var percentageLetter = function(s, letter) {
       let count=0;
    for(let i=0;i<s.length;i++)
        {
            if(letter===s[i])
                {
                    count++;
                }
        }
     let p=count*100;
    return Math.floor(p/s.length);
};